src/org/usfirst/frc/team930/robot/Robot.java

```diff
@@ -5,56 +5,78 @@
 import edu.wpi.first.wpilibj.IterativeRobot;
 import edu.wpi.first.wpilibj.command.Command;
 import edu.wpi.first.wpilibj.command.Scheduler;
+import edu.wpi.first.wpilibj.command.Subsystem;
 import edu.wpi.first.wpilibj.livewindow.LiveWindow;
 
-import org.usfirst.frc.team930.robot.commands.Drive;
-import org.usfirst.frc.team930.robot.commands.ExampleCommand;
+import org.usfirst.frc.team930.robot.commands.CloseClaw;
+import org.usfirst.frc.team930.robot.commands.CloseLeftClaw;
+import org.usfirst.frc.team930.robot.commands.CloseRightClaw;
+import org.usfirst.frc.team930.robot.commands.OpenClaw;
+import org.usfirst.frc.team930.robot.commands.OpenLeftClaw;
+import org.usfirst.frc.team930.robot.commands.OpenRightClaw;
 import org.usfirst.frc.team930.robot.subsystems.Arm;
-import org.usfirst.frc.team930.robot.subsystems.Claw;
 import org.usfirst.frc.team930.robot.subsystems.Drivetrain;
-import org.usfirst.frc.team930.robot.subsystems.ExampleSubsystem;
-
+import org.usfirst.frc.team930.robot.subsystems.Claw;
 
 public class Robot extends IterativeRobot {
 
+	
+	
 	public static final Arm arm = new Arm();
-	public static final Claw claw = new Claw();
+	public static final Claw leftClaw = new Claw(Claw.leftRelay, Claw.leftOpen, Claw.leftClosed, 1);
+	public static final Claw rightClaw = new Claw(Claw.rightRelay, Claw.rightOpen, Claw.rightClosed, 2);
 	public static final Drivetrain drivetrain = new Drivetrain();
 	public static OI oi;
 
-    Command drive;
+	Command closeClaw;
```

Comment:
The closeClaw and openClaw would only be used if we wanted to open both claws simultaneously through a single button.